docs/source/autoscaling.rst

```diff
@@ -120,40 +96,20 @@ The currently available decicion policies are:
   :offset:
     Float between 0.0 and 1.0, representing expected baseline load for each container.
     Defaults to 0.0.
-  :forecast_policy:
-    See "Forecast policies" below.
   :good_enough_window:
```

Comment:
should we mention that this isn't actually supported yet?
```suggestion
  :good_enough_window (not supported in Kubernetes):
```
or just go ahead and delete this section for now until we do?

Reply:
good catch, I'll reword this.